### PR TITLE
chore(cli): add skeleton for "env upgrade" command

### PR DESCRIPTION
**Pull request by efekarakus**
_By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice._

### CODE REVIEW

File: internal/pkg/cli/flag.go [hunk 1]
```diff
@@ -210,4 +211,6 @@ Accepts valid Go duration strings. For example: "2h", "1h30m", "900s".`
 	scheduleFlagDescription = `The schedule on which to run this job. 
 Accepts cron expressions of the format (M H DoM M DoW) and schedule definition strings. 
 For example: "0 * * * *", "@daily", "@weekly", "@every 1h30m".`
+
+	upgradeAllEnvsDescription = "Optional. Upgrade all environments."
```
Comment (huanjani) [inline internal/pkg/cli/flag.go:215]: Oh! That's pretty cool.

**Review by huanjani — APPROVED** [review]
LGTM!

File: internal/pkg/cli/env_upgrade.go [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+	"github.com/spf13/cobra"
+)
+
+// envUpgradeVars holds flag values.
+type envUpgradeVars struct {
+	*GlobalOpts
+
+	name string // Required. Name of the environment.
+	all  bool   // True means all environments should be upgraded.
```
Comment (bvtujo) [inline internal/pkg/cli/env_upgrade.go:17]: Does `all` supersede `name`? For example, will we require a customer to run `copilot env upgrade --all --name test` if they want to upgrade everything?

**Review by bvtujo — APPROVED** [review]
This looks good to me! Just a question

File: internal/pkg/cli/env_upgrade.go (hunk 2 above)
Comment (efekarakus) [reply internal/pkg/cli/env_upgrade.go:17]: Yeah that was my thinking too, so `copilot env upgrade --all` to upgrade everything. They won't be able to use `--name` and `--all` together, `--name` will be to upgrade only a specific env.

**Pull request merged**